clang/lib/CodeGen/Targets/AArch64.cpp

```diff
@@ -857,6 +864,36 @@ void AArch64TargetCodeGenInfo::checkFunctionCallABI(
           << Callee->getDeclName();
 }
 
+void AArch64ABIInfo::appendAttributeMangling(TargetClonesAttr *Attr,
+                                             unsigned Index,
+                                             raw_ostream &Out) const {
+  appendAttributeMangling(Attr->getFeatureStr(Index), Out);
+}
+
+void AArch64ABIInfo::appendAttributeMangling(StringRef AttrStr,
+                                             raw_ostream &Out) const {
+  if (AttrStr == "default") {
+    Out << ".default";
+    return;
+  }
+
+  Out << "._";
+  SmallVector<StringRef, 8> Features;
+  AttrStr.split(Features, "+");
+  for (auto &Feat : Features)
+    Feat = Feat.trim();
+
+  // TODO Lexicographical order won't break the ABI if priorities change.
```

Comment:
Usually we do a 'FIXME' if we are going to let this sit a while (allows folks to use 'TODO' for personal use).

Also, I don't really get what this is saying, so if this is going to stick around, we probably need to clarify it for future visitors.

Reply:
I can change this to say FIXME instead and write a more descriptive comment. For context please read https://github.com/llvm/llvm-project/pull/79316#discussion_r1469932618. It shouldn't stick around for long, as soon as we reach a conclusion I can update the sorting criteria of this lambda.

Reply:
SGTM!